src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Bedrock/InvokeModelAsyncWrapper.cs

```diff
@@ -73,7 +77,14 @@ void InvokeTryProcessResponse(Task<InvokeModelResponse> responseTask)
             }
         }
 
-        private void ProcessInvokeModel(ISegment segment, InvokeModelRequest invokeModelRequest, InvokeModelResponse invokeModelResponse, IAgent agent)
+        private static object GetTaskResult(object task)
```

Comment:
Nice!

Reply:
I'm an expert-level copy-and-paste coder!